javacutil/src/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -336,8 +337,11 @@ public static Tree getAssignmentContext(final TreePath treePath) {
                 (node instanceof NewArrayTree) ||
                 (node instanceof NewClassTree) ||
                 (node instanceof ReturnTree) ||
-                (node instanceof VariableTree))
+                (node instanceof VariableTree)) {
             return node;
+        } else if (node instanceof ConditionalExpressionTree) {
```

Comment:
(I mean changing the return statement instead of adding the branch for the `ConditionalExpressionTree`)